### DISCUSSION TITLE
Request for Single HTML File Output

### DISCUSSION THREAD

**Discussion by zpolat** [WasiqB/multiple-cucumber-html-reporter#268] [Ideas]
I hope this message finds you well. First of all, I would like to express my appreciation for your work on the Multiple-Cucumber-HTML-Reporter tool. It has proven to be extremely valuable for my team's testing efforts.

We are currently exploring ways to simplify our report distribution process. To this end, I am reaching out to inquire about the possibility of generating a single HTML file report with the Multiple-Cucumber-HTML-Reporter. More specifically, we're interested in a format that doesn't require the additional assets and features subfolders, thus enabling the entire report to be contained within a single HTML file.

I understand that this might not be a common use case, and it may also introduce certain complexities, as it would require inlining all CSS and JavaScript assets into the HTML file. However, such a feature would greatly simplify our workflow.

I would like to know if this feature is currently available, or if it might be feasible to implement it in the future. I would be more than happy to contribute to the discussion on how this might be best achieved, if that would be helpful.

**Comment by WasiqB** [comment]
Hi @zpolat ,

Thanks for sharing this idea. I will work on this feature in upcoming v4.1.0.

For now, I would like to request you to try out the new version of [4.0.0-beta.0](https://github.com/WasiqB/multiple-cucumber-html-reporter/releases/tag/v4.0.0-beta.0) which has been released yesterday.

Share your thoughts about the new beta version.
